examples/data-sources/bloxone_dns_a_records/data-source.tf

```diff
@@ -0,0 +1,16 @@
+# Get "A" records filtered by an attribute
+data "bloxone_dns_a_records" "example_by_attribute" {
+  filters = {
+    "absolute_name_spec" = "abc.example.com"
```

Comment:
Do we need the quotes around the 'key'?

Reply:
It is not needed, but even if it is present, it is not a problem.